### PR TITLE
Preparing content for activator template

### PR DESCRIPTION
**Pull request by fedefernandez**
This PR includes the necessary data for Activator template

Please @raulraja could you review? Thanks

### CODE REVIEW

**Comment by fedefernandez** [comment]
Even the main purpose of this PR was prepare to activator template I've fixed problem with copyright on `build.sbt` and added _proguard_ configuration. Please, @raulraja could you review? Thanks

**Comment by fedefernandez** [comment]
Also, someone with _administrator_ permission on this repo should rename it to **scala-on-android-preparing-the-environment**

**Comment by nickelsberry** [comment]
@fedefernandez Done, can you add an appropriate readme file to the repo as well.

File: activator.properties [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name=scala-android-preparing-the-environment
+title=Scala in Android - Preparing the environment
+description=A starter application with Scala and Android.
+tags=scala,android
+authorLogo=https://avatars1.githubusercontent.com/u/5580770
```
Comment (raulraja) [inline activator.properties]: We should use a permalink from our site so we can easily change assets down the road. I tasked @rafaparadela on creating a process for it that will get added to the web README

File: tutorial/index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  -->
+
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Scala in Android: Preparing the Environment</title>
+</head>
+<body>
+<div>
+    <p>You've just created Android Project with Scala!</p>
```
Comment (raulraja) [inline tutorial/index.html]: `Welcome to Scala on Android!` Vs `You've just created Android Project with Scala!`

**Comment by fedefernandez** [comment]
@raulraja comments addressed. Please review in order to submit the activator template. Thanks

**Comment by raulraja** [comment]
LGTM

**Pull request merged**